### PR TITLE
ClusterLoader - Adding measurement dispose

### PR DESCRIPTION
**Pull request by krzysied**
Adds `Dispose` method to all measurements to ensure disposing allocated resources.

### CODE REVIEW

File: clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go [hunk 1]
```diff
@@ -164,6 +166,10 @@ func (g *ContainerResourceGatherer) StartGatheringData() {
 // It returns an error if the resource usage at any percentile is beyond the
 // specified resource constraints.
 func (g *ContainerResourceGatherer) StopAndSummarize(percentiles []int, constraints map[string]util.ResourceConstraint) (*ResourceUsageSummary, error) {
+	if !g.isRunning {
+		glog.Errorf("ContainerResourceGatherer is not running")
```
[the remark this answers is not in the document]
Comment (krzysied) [reply clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go]: Done.

File: clusterloader2/pkg/measurement/common/pod_startup_latency.go [hunk 2]
```diff
@@ -147,6 +157,7 @@ func (p *podStartupLatencyMeasurement) gather(c clientset.Interface) ([]measurem
 	schedToWatchLag := make([]time.Duration, 0)
 	e2eLag := make([]time.Duration, 0)
 
+	p.isRunning = false
```
[the remark this answers is not in the document]
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/pod_startup_latency.go]: Done.
Although I don't like to call `Dispose()` in the middle of gather function. I've added `stop` function. This function will be called in gather and in dispose method.

File: clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go [hunk 3]
```diff
@@ -246,3 +252,11 @@ func (g *ContainerResourceGatherer) StopAndSummarize(percentiles []int, constrai
 	}
 	return &summary, nil
 }
+
+// Stop stops stat gathering workers.
+func (g *ContainerResourceGatherer) Stop() {
```
[the remark this answers is not in the document]
Comment (krzysied) [reply clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go]: Done.

File: clusterloader2/api/extensions.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
+/*
```
Comment (wojtek-t) [inline clusterloader2/api/extensions.go]: Why is this in this file? Isn't it already merged?

Comment (krzysied) [reply clusterloader2/api/extensions.go]: It is merged. After rebasing this PR, it looks like it's fixed.
I'm not sure why this happened.

**Pull request merged**